ush/user.jsonschema

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
How is this file used in relation to `experiment.jsonschema`? This isn't described in the documentation.

Reply:
Hmmm. Let me try to find where it should live in the docs. It checks only the user-supplied YAML to make sure they've provided the required information. I think `account` is the only other thing that is technically "required", but not every platform does actually require an account, so I don't know how to handle that with a schema.